proc_scripts/phasing.py

```diff
@@ -201,12 +201,12 @@ def hermitian_function_test(s, down_from_max=0.5, rel_shift=1.5,shift_points=120
     data_for_peak /= max_val
     pairs = data_for_peak.contiguous(lambda x: abs(x) >
             down_from_max)
-    peak_location = pairs[0,:] #gives the longest pair
-    peak_center = peak_location.mean()
+    peak_bounds = pairs[0,:] #gives the longest pair
+    peak_center = data_for_peak['t2':(0.1e-3,None)].argmax('t2').item()
     s.setaxis('t2',lambda x: x-peak_center)
     s.register_axis({'t2':0})
     logging.debug(strm("closest to 0", s.getaxis('t2')[np.argmin(abs(s.getaxis('t2')-0))]))
-    max_shift = np.diff(peak_location).item()/2
+    max_shift = np.min(abs(peak_center-peak_bounds))
```

Comment:
wouldn't this return two values? or does the min take care of that?

Reply:
the latter

Reply:
so to be clear. we are saying take the minimum of the two values that are returned when we take the difference between the peak bounds and the center?

Reply:
yes